types/shims-vue.d.ts

```diff
@@ -1,8 +1,5 @@
-import { NuxtI18nSeo } from 'nuxt-i18n/types/nuxt-i18n'
-
 declare module 'vue/types/vue' {
```

Comment:
```suggestion
export {}
declare module 'vue/types/vue' {
```
ここ、1行目に `export {}` 追加お願いします（詳しい説明は割愛しますが、1行目のやつが無いと Vue への inject 関連がうまく動かないのです）